WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -40346,13 +40365,22 @@ e_latin_empire = {
 	capital = 496 # Byzantion
 	
 	short_name = yes
+	dynasty_title_names = no
 	
 	
 	allow = {
 		OR = {
+			has_romance_culture_trigger = yes
+			culture_group = latin
```

Comment:
You didn't read all the code. Directly below, it explicitly states that normal Greeks and Romans cannot form the Latin Empire.